include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -3078,30 +3081,30 @@ namespace internal
           Number *      out = out_array + d * dofs_per_component;
           // Need to select 'apply' method with hessian slot because values
           // assume symmetries that do not exist in the inverse shapes
-          evaluator.template hessians<0, false, false>(in, out);
+          evaluator.template hessians<0, true, false>(in, out);
```

Comment:
Note that the old implementation stored the transpose of the variant I provide now; the reason is that for generalization to the non-square (or pseudo inverse) case it is natural to combine the operations with the rows and columns switched.